### ISSUE TITLE
RandomChat.GetEvent() blocking

### ISSUE THREAD

**Issue by NickDomanti** [Coal0/python-omegle#7]
Calling GetEvent() on an instance of RandomChat freezes the code completely, it proceeds no more. Probably because something in Omegle changed since the last time you updated this.

**Comment by JonZavialov** [comment]
I'm getting this error too.

**Comment by mrseeker** [comment]
It seems that the "blocking" comes from Omegle waiting for a response... which never seems to happen.

**Comment by NickDomanti** [comment]
> It seems that the "blocking" comes from Omegle waiting for a response... which never seems to happen.

Yep, that's what I thought. Omegle folks must have changed how their system works since this library got updated the last time.

**Comment by mrseeker** [comment]
What you actually getting is an error response and a notification to update the webapp, but I haven't been able to replicate the exact stuff they are looking for in the headers.

**Comment by NickDomanti** [comment]
> What you actually getting is an error response and a notification to update the webapp, but I haven't been able to replicate the exact stuff they are looking for in the headers.

How did you find out?

**Comment by mrseeker** [comment]
I know kung-fu?

But jokes aside, I tried the came command in curl and it came back with an error message.

**Comment by NickDomanti** [comment]
oh yea right.
